pkgs/fake_async/CHANGELOG.md

```diff
@@ -1,13 +1,21 @@
 ## 1.3.3-wip
 
+* Make the zone `create*Timer` and `scheduleMicrotask`
+  be responsible for running callbacks in the zone they're
+  scheduled in, matching (new) standard zone behavior.
+  (The `Timer` constructors and top-level `scheduleMicrotask`
+  used to bind their callback, but now only registers it,
+  leaving the zone to run in the correct zone and handle errors.)
+* Make periodic timers increment their `tick` by more than one
+  if `elapseBlocking` advanced time past multiple ticks.
+
 ## 1.3.2
 
 * Require Dart 3.3
 * Fix bug where a `flushTimers` or `elapse` call from within
   the callback of a periodic timer would immediately invoke
   the same timer.
 * Move to `dart-lang/test` monorepo.
-* Require Dart 3.5.
```

Comment:
Didn't actually change `pubspec.yaml`.
(Only one of the first and last line here is correct and needed.)